wallet/test/integration/configuration.yaml

```diff
@@ -22,10 +22,10 @@ default: &default
           seed: 0
         blockVersionData: &default_core_genesis_spec_blockVersionData
           scriptVersion:     0
-          slotDuration:      10000
+          slotDuration:      5000
```

Comment:
Whoops :man_shrugging: . This wasn't purely necessary and ended up in a commit by accident; nevertheless...... it'll make the test run slightly faster without any foreseen drawbacks so, why not.